packages/ui/src/elements/contexts/index.tsx

```diff
@@ -128,8 +128,9 @@ export type FlowMetadata = {
     | 'complete'
     | 'accountSwitcher'
     | 'chooseOrganization'
+    | 'chooseWallet'
     | 'enterpriseConnections'
-    | 'chooseWallet';
+    | 'organizationCreationDisabled';
```

Comment:
should we keep the keys similar for a group of flows e.g. `chooseOrganizationDisabled` vs `organizationCreationDisabled`?

Reply:
I think `choose` could be confusing here, as this is based on the `creation` action - eg: if the user has existing memberships to choose, then they wouldn't see that flow step